data/json/items/comestibles/mutagen.json

```diff
@@ -156,16 +161,16 @@
     "copy-from": "iv_mutagen_flavor",
     "type": "COMESTIBLE",
     "name": "lupine serum",
-    "description": "A super-concentrated mutagen strongly resembling blood.  You need a syringe to inject it...  if you really want to?",
-    "color": "red",
+    "description": "A super-concentrated mutagen as white as a full moon.  You need a syringe to inject it...  if you really want to?",
+    "color": "white",
     "use_action": { "type": "mutagen_iv", "mutation_category": "LUPINE" }
   },
   {
     "id": "iv_mutagen_medical",
     "copy-from": "iv_mutagen_flavor",
     "type": "COMESTIBLE",
     "name": "medical serum",
-    "description": "A super-concentrated substance.  Judging by the amount, it should be delivered via injection.  You'd need a syringe.",
+    "description": "A super-concentrated mutagenic substance.  Judging by the amount, it should be delivered via injection.  You'd need a syringe.",
```

Comment:
I would bring this in line with the others, don't see a good reason to have it be the only different one.